test/SearchTest/UsableTait/OptionsTraitTest.php

```diff
@@ -20,27 +20,33 @@
 
 namespace oat\search\test\searchTest\UsableTrait;
 
+use oat\search\test\UnitTestHelper;
+
 /**
  * Description of OptionsTraitTest
  *
  * @author Christophe GARCIA <christopheg@taotesting.com>
  */
-class OptionsTraitTest  extends \oat\search\test\UnitTestHelper {
-    
+class OptionsTraitTest extends UnitTestHelper
+{
+
     protected $instance;
-    
-    public function setup() {
-        
+
+    public function setup(): void
+    {
+
```

Comment:
Can we remove some extra lines inserted in the files?

Reply:
done